__tests__/app.test.js

```diff
@@ -198,6 +198,72 @@ describe("/api/articles", () => {
         });
       });
   });
+  test("GET:200 When provided a query sort_by follwed by a valid column name (author) sorts the array by that column name", () => {
+    return request(app)
+      .get("/api/articles?sort_by=author")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("author", {
+          descending: true,
+        });
+      });
+  });
+  test("GET:200 When provided a query sort_by follwed by a valid column name (article_id) sorts the array by that column name", () => {
+    return request(app)
+      .get("/api/articles?sort_by=article_id")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("article_id", {
+          descending: true,
+        });
+      });
+  });
```

Comment:
article_id doesn't really make sense to have as a valid sort_by, because it's more used for the server behind the scenes as opposed to being something that's of interest to the user